src/dispatch.rs

```diff
@@ -121,7 +121,7 @@ impl std::fmt::Display for MonitorIdentifier<'_> {
     fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
         let out = match self {
             MonitorIdentifier::Direction(dir) => dir.to_string(),
-            MonitorIdentifier::Id(id) => id.to_string(),
+            MonitorIdentifier::Id(id) => id.map_or("None".to_string(), |id| id.to_string()),
```

Comment:
I don't think this is correct, as this is passed into a hyprland dispatcher which does not take None as an input.

Reply:
Wouldn't the dispatcher call simply fail (which would be expected behaviour)? 

For example if I run `hyprctl dispatch focusmonitor None`, the command returns `ok` but nothing actually happens, because I don't have a monitor with that id. If this library calls focusmonitor on a "None" string, wouldn't the same thing occur?

Reply:
I think it would be better to only make it an option (`Option<MonitorId>`) in the actual data structs inorder not to interfere with other modules.

Reply:
Sorry for chiming in, but I think this particular change was done because an `Option` can't be formatted just like that as it doesn't have the `to_string` method. So a fallback for when it's `None` has to be put in place here. How about using `null` instead as this is what Hyprland itself reports?

Reply:
Done. @chmanie can you test this solution?

Reply:
@chmanie Oops, didn't see your message until after I refreshed the page. I thought @yavko meant that the reason it panics is because the `null` received by Hyprland when getting clients and workspaces can't be parsed by their corresponding data structures. My original commit only changed the MonitorIdentifier, can that ever have a null ID? Is it possible to have a monitor with id=null when you call `hyprctl monitors`?

Reply:
Ok, sorry, I was confised, please ignore me. The latest changes look perfectly fine to me :pray: